numpy/linalg/linalg.py

```diff
@@ -1615,24 +1632,25 @@ def pinv(a, rcond=1e-15 ):
 
     Parameters
     ----------
-    a : (M, N) array_like
-      Matrix to be pseudo-inverted.
-    rcond : float
-      Cutoff for small singular values.
-      Singular values smaller (in modulus) than
-      `rcond` * largest_singular_value (again, in modulus)
-      are set to zero.
+    a : (..., M, N) array_like
+        Matrix to be pseudo-inverted. The ability to handle stacked
+        matrices was added in Numpy 1.13.0.
+    rcond : (...) arr_like of float
+        Cutoff for small singular values.
+        Singular values smaller (in modulus) than
```

Comment:
How about "less than" rather than smaller. The singular values are always non-negative reals.